### PR TITLE
release Mixxx 2.1.4

### PR DESCRIPTION
**Pull request by Be-ing**
Fix track selection not getting shown in the track
table on Windows. There are no changes to the
source code, but the Jenkins build configuration
was changed to delete the Jenkins workspace before
each build.
https://bugs.launchpad.net/mixxx/+bug/1751482

### CODE REVIEW

**Comment by sblaisot** [comment]
no change to README.md nor build/debian/changelog this time ?

**Comment by Be-ing** [comment]
Whoops, good catch.

**Comment by Pegasus-RPG** [comment]
Even for bugfix releases we need a checklist.

**Comment by Be-ing** [comment]
Yes, I'll work on writing that out soon.

File: build/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+mixxx (2.1.3-0ubuntu1) bionic; urgency=medium
```
Comment (trebmuh) [inline build/debian/changelog]: @Be-ing : shouldn't it be 2.1.4-0ubuntu1 here?

**Comment by daschuer** [comment]
Don't forget to push and pull from transiflex.

**Comment by Be-ing** [comment]
Is that documented anywhere? I don't know how to do that. Have there been any changes anyway?

File: build/debian/changelog (hunk 1 above)
Comment (Be-ing) [reply build/debian/changelog]: good catch, thanks. Fixed in a force pushed commit.

**Comment by Be-ing** [comment]
I ran `tx pull` and there are no translation updates.

**Pull request merged**